site/book/07-effective-customizations/01-single-value-replacement.md

```diff
@@ -0,0 +1,26 @@
+## Scenario
+
+I have a single value replacement in my package, I don’t want package consumers to look through all the yaml files to find the value I want them to set, it seems easier to just create a parameter for this value and have the user look at Kptfile for inputs.
```

Comment:
single value to replace

Reply:
This is a long run-on sentence. 

I suggest someone should just proofread this whole PR.

Reply:
I'll break up this sentence but once we are OK with general content I can also loop in Nicole to proof read.  English is my second language 😄